.github/workflows/ci.yml

```
@@ -107,7 +107,10 @@ jobs:
 
       - run: cargo install --path crates/cargo-codspeed --locked
 
-      - run: cargo codspeed build -p ${{ matrix.package }}
+      - run: |
+          # Remove the cargo config else it forces instrumentation mode
+          rm .cargo/config.toml || true
```

might be more conventional
```suggestion
          rm -f .cargo/config.toml
```